lib/ansible/cli/__init__.py

```diff
@@ -70,6 +70,13 @@ def __init__(self, args, callback=None):
         self.parser = None
         self.callback = callback
 
+        if C.DEVEL_WARNING and __version__.endswith('dev0'):
+            display.warning(
+                'You are running the development version of Ansible. You should only run Ansible from "devel" if '
+                'you are actively developing content for Ansible. This is a rapidly changing source of code and can '
+                'become unstable at any point.'
```

Comment:
Do we want/need to add a mention to DEVEL_WARNING here?

Reply:
I didn't add it for users.  It would not have been added, had it not broken so many tests.  I added it to allow CI to prevent it, so I didn't have to rewrite a bunch of tests.